util/configure

```diff
@@ -697,6 +701,14 @@ _END_
             $luajit_xcflags .= " -DLUAJIT_ENABLE_LUA52COMPAT";
         }
 
+        if (!$no_luajit_gc64
+            && `uname -a` =~ /\bx86_64\b/
```

Comment:
Note: iirc, the GC64 mode is required on ARM64.

Reply:
I think we should use Perl's own architecture information instead of invoking `uname` which is not available everywhere.

BTW, GC64 is required on ARM64 so we do not need to worry about that at all. The LuaJIT build system enables that automatically anyway.

Reply:
Should we then update existing invocations of `uname`? i.e. https://github.com/openresty/openresty/blob/feb1bc4c2a060abf48c24a5cdbe685f156e80829/util/configure#L325

Reply:
We could though it's less an issue since it first makes sure it is on darwin.

Reply:
Updated

Reply:
> The LuaJIT build system enables that automatically anyway.

That's helpful, thanks, I was about to get my hands on an ARM64 box to test this (still planning on doing so today or tomorrow).

Reply:
@thibaultcha Great. Thanks! It would be awesome if you could run the tests on 32-bit arm systems at the same time, to cover corner cases as in openresty/luajit2#37.